src/QuantumSdk/Sdk/Sdk.targets

```
@@ -35,9 +35,9 @@
   <Target Name="QSharpClean" BeforeTargets="Clean">
     <ItemGroup>
       <_FilesToClean Include="$(GeneratedFilesOutputPath)**" />
-      <Compile Remove="$(_FilesToClean)**/*.g.cs" />
+      <Compile Remove="$(_FilesToClean)**/*.cs" />
     </ItemGroup>
-    <Delete Files="@(_FilesToClean)" />
+    <Delete Files="$(GeneratedFilesOutputPath)**/*.*" />
```

Was there a reason for changing that? I think removing _FilesToClean should be fine.
```suggestion
    <Delete Files="@(_FilesToClean)" />
```

---

@anpaz had suggested that the `<path>/**` format stored in _FilesToClean was unsafe and recommended that it be changed to `<path>/**/*.*` Since I'm not clear on what all this is, if you still think I made a mistake and it should be _FilesToClean instead, just respond to this thread again and let me know.

---

Actually now that I take a closer look at this, it _does_ seem like I would want the `<path>/**` syntax here, because we would want to remove any directories as well. I will make the change.

---

done